src/mlpack/methods/reinforcement_learning/replay/episodic_replay.hpp

```diff
@@ -0,0 +1,271 @@
+#ifndef MLPACK_METHODS_RL_REPLAY_EPISODIC_REPLAY_HPP
+#define MLPACK_METHODS_RL_REPLAY_EPISODIC_REPLAY_HPP
+
```

Comment:
Can you add a simple description here, you can take a look at the exsisting files for examples.